core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -244,4 +244,13 @@ private TableProperties() {
 
   public static final String UPSERT_MODE_ENABLE = "write.upsert.enable";
   public static final boolean UPSERT_MODE_ENABLE_DEFAULT = false;
+
+  public static final String REWRITE_PARTIAL_PROGRESS_ENABLED = "rewrite.partial-progress.enabled";
+  public static final String REWRITE_PARTIAL_PROGRESS_MAX_COMMITS = "rewrite.partial-progress.max-commits";
+  public static final String REWRITE_MAX_FILE_GROUP_SIZE_BYTES = "rewrite.max-file-group-size-bytes";
```

Comment:
These are still bin-pack options, right? Sort is a subtype of bin packing. My concern is that other strategies may not respect these.

Reply:
these are options defined in `RewriteDataFiles` https://github.com/apache/iceberg/blob/master/api/src/main/java/org/apache/iceberg/actions/RewriteDataFiles.java, that's why I prefixed with `rewrite`.